packages/core/src/modules/proofs/formats/dif-presentation-exchange/DifPresentationExchangeProofFormatService.ts

```diff
@@ -354,27 +356,38 @@ export class DifPresentationExchangeProofFormatService
 
   public async getCredentialsForRequest(
     agentContext: AgentContext,
-    { requestAttachment }: ProofFormatGetCredentialsForRequestOptions<DifPresentationExchangeProofFormat>
+    { requestAttachment, proofFormats }: ProofFormatGetCredentialsForRequestOptions<DifPresentationExchangeProofFormat>
   ) {
     const ps = this.presentationExchangeService(agentContext)
     const { presentation_definition: presentationDefinition } =
       requestAttachment.getDataAsJson<DifPresentationExchangeRequest>()
 
     ps.validatePresentationDefinition(presentationDefinition)
 
-    const presentationSubmission = await ps.getCredentialsForRequest(agentContext, presentationDefinition)
+    // Set default values
+    const { filterByNonRevocationRequirements = true } = proofFormats?.presentationExchange ?? {}
+    const presentationSubmission = await ps.getCredentialsForRequest(agentContext, presentationDefinition, {
+      filterByNonRevocationRequirements,
+    })
     return presentationSubmission
   }
 
   public async selectCredentialsForRequest(
     agentContext: AgentContext,
-    { requestAttachment }: ProofFormatSelectCredentialsForRequestOptions<DifPresentationExchangeProofFormat>
+    {
+      requestAttachment,
+      proofFormats,
+    }: ProofFormatSelectCredentialsForRequestOptions<DifPresentationExchangeProofFormat>
   ) {
     const ps = this.presentationExchangeService(agentContext)
     const { presentation_definition: presentationDefinition } =
       requestAttachment.getDataAsJson<DifPresentationExchangeRequest>()
 
-    const credentialsForRequest = await ps.getCredentialsForRequest(agentContext, presentationDefinition)
+    // Set default values
+    const { filterByNonRevocationRequirements = true } = proofFormats?.presentationExchange ?? {}
+    const credentialsForRequest = await ps.getCredentialsForRequest(agentContext, presentationDefinition, {
+      filterByNonRevocationRequirements,
+    })
```

Comment:
Will it work if we make a presentation that does not adhere to non-revocation requirements?